src/main/resources/templates/gcc/parent-info-review.html

```diff
@@ -16,6 +16,8 @@
                             th:text="#{parent-info-review.personal.information}"></strong></p>
                     <ol class="list--bulleted">
                         <li th:text="|${inputData.get('parentFirstName')} ${inputData.get('parentLastName')}|"></li>
+                        <li th:if="${!#strings.isEmpty(inputData.get('parentPreferredName'))}"
+                            th:text="|#{parent-info-review.preferred.name}: ${inputData.get('parentPreferredName')}|"></li>
```

Comment:
we only need the preferred name here
```suggestion
                            th:utext="${inputData.get('parentPreferredName')}"></li>
```